src/data/behandlerdialog/behandlerMeldingTexts.ts

```diff
@@ -56,3 +56,7 @@ export const returLegeerklaringTexts = {
       "Dersom du allerede har sendt inn regning for den mangelfulle erklæringen, forutsetter vi at det ikke blir sendt regning for ny utfylt Legeerklæring ved arbeidsuførhet.",
   },
 };
+
+export const meldingFraNAVTexts = {
+  header: "Melding fra NAV",
+};
```

Comment:
Vi får gi Stine/TH/Astrid en preview av hvordan dette blir, og høre hva de synes 👍🏼

Reply:
Både TH og Stine har fått sett hvordan det ser ut og godkjent, så vi er good to go der 🙏